INSTALL

```diff
@@ -394,6 +394,9 @@
                    Don't compile in filename and line number information (e.g.
                    for errors and memory allocation).
 
+  no-fips
+                   Don't compile the FIPS module
+
```

Comment:
I'm not sure it should generate the `OPENSSL_NO_FIPS` macro, though.  That will just add confusion, 'cause that's not a library option.

There's this section in `Configure` that has a selection of config options that shouldn't end up as C macros of that form, `'fips'` should be added there:

https://github.com/openssl/openssl/blob/e321ba28e89d40ab750e353d5bd7fc405e9c2761/Configure#L1224-L1226

Reply:
> I'm not sure it should generate the OPENSSL_NO_FIPS macro, though. That will just add confusion, 'cause that's not a library option.

I guess I don't understand that argument. It's something that we want to be able to write conditional code on (as evidenced by the fact I've used it already in the tests) - so why would we suppress the macro generation?

Reply:
Hmmm. Thinking about this more - I guess this is about whether it is appropriate to have the macro in opensslconf.h (a public header file). Is that it?

Reply:
Yes, exactly